app/templates/compare_revisions.html

```diff
@@ -63,27 +63,19 @@
     <div class="grid-row">
       <div class="column-two-thirds page-section">
         <p>
-          {% if all_update_events is none %}
-            Multiple
+          {% if latest_update_events %}
+            {% if min_number_of_users_who_made_edits > 1 %}
+              More than one user has edited this service. The last user to edit this page was
+            {% endif %}
+            {{ latest_update_events.0.user }}
```

Comment:
I find the interleaving of messages here (to avoid repetition of `{{ latest_update_events.0.user }}` in the template) quite difficult to follow.  I think something like this would make it easier to see what's going on with the different possible cases:
```
<p>
  {% if latest_update_events %}

    {% if min_number_of_users_who_made_edits > 1 %}
      More than one user has edited this service.
      The last user to edit this service was {{ latest_update_events.0.user }}
      on {{ latest_update_events[0].createdAt|dateformat }}
    {%- endif -%}

    {% if min_number_of_users_who_made_edits == 1 %}
      {{ latest_update_events.0.user }} made {{ (all_update_events or (oldest_update_events+latest_update_events))|length }}
      {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") }}
      {% if latest_update_events[0].createdAt|dateformat == oldest_update_events[-1].createdAt|dateformat %}
        on {{ latest_update_events[0].createdAt|dateformat }}
      {%- else %}
        between {{ oldest_update_events[-1].createdAt|dateformat }}
        and {{ latest_update_events[0].createdAt|dateformat }}
      {%- endif -%}
    {%- endif -%}

  {% else %}
    0 unapproved edits
  {%- endif -%}
  .
</p>
```

Reply:
This looks very sensible :)